build.cake

```diff
@@ -207,6 +213,7 @@ Task("Publish-NuGet")
     .WithCriteria(() => parameters.IsMainCakeRepo)
     .WithCriteria(() => parameters.IsMainCakeBranch)
     .WithCriteria(() => parameters.IsTagged)
+    .ContinueOnError()
```

Comment:
I think that `ContinueOnError` will suppress the `OnError` handling but I'm not sure.